assets/css/defaults.css

```diff
@@ -60,16 +72,13 @@ td:hover {
 }
 .headerTitle {
 	margin-top: 10px;
-	font-family: simpleBattleHeader;
+	font-family: simpleBattleHeader, fantasy;

```

Comment:
In case of the font not loading properly, always give a fallback font that  feat the type of font you applied